client/src/index.jsx

```diff
@@ -33,47 +33,49 @@ class Player extends React.Component {
 
   // }
 
-  initialize() {
+
+  componentDidMount() {
     $.ajax({
       type: "GET",
-      url: `http://52.37.102.63:3005/songdata/${this.songId}`,
+      url: `http://localhost:3005/songdata/${this.song_id}`,
       success: (res) => {
```

Comment:
Consider `{ data }` instead of `res` here for readability.

Reply:
Can you please expand on that? I'm not sure I follow your reasoning here.